src/reducers/nodes.js

```diff
@@ -59,9 +59,8 @@ function nodeReducer(nodeState = {}, action) {
         parameters: Object.assign({}, nodeState.parameters, {
           [id]: data.parameters,
         }),
-        // the data returned from the API under the field name 'type' for dataset type nodes
-        datasetType: Object.assign({}, nodeState.datasetType, {
```

Comment:
Is this feature now removed or replaced then?

Reply:
indeed it does seem strange to remove `datasetType` nodes as it still exists with `plot` type as an additional subset of dataset nodes?

Reply:
I am still confused as to why `datasetType` is removed here as I recall this field is particularly returned for `data` type nodes ( which `data` type node response object with the `type` field, it is different from the general node.type field that we use within the app.) 

I just checked and can confirm that the response object still returns the field `type` for data objects ( see below), hence there is a need to keep the `datasetType` field. 
![image](https://user-images.githubusercontent.com/27775658/119826749-442a8d80-bef0-11eb-9415-8c0bd468474f.png)